sql-plugin/src/main/311until340-nondb/scala/com/nvidia/spark/rapids/shims/ShimDataSourceV2ScanExecBase.scala

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright (c) 2022, NVIDIA CORPORATION.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.nvidia.spark.rapids.shims
-
-import org.apache.spark.sql.execution.datasources.v2.DataSourceV2ScanExecBase
-
-trait ShimDataSourceV2ScanExecBase extends DataSourceV2ScanExecBase
```

Comment:
There's a 311+-db version of this that is now unused?  Seems like we would need it just for the `computeStats` override.